code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -4113,6 +4114,44 @@
 	mob2spawnA = /mob/living/simple_animal/hostile/bigroach
 	mob2spawnB = /mob/living/simple_animal/hostile/bigroach/queen //greater odds than getting a queen via mutating roaches (0.5%)
 
+/datum/chemical_reaction/aminocorydon
+	name = "Aminocorydon"
+	id = AMINOCORYDON
+	result = AMINOCORYDON
+	required_reagents = list(AMINOMICIN = 1, BANANA = 5)
+	result_amount = 1
+
+/datum/chemical_reaction/aminocorydon/required_condition_check(datum/reagents/holder)
+	if(istype(holder.my_atom, /obj/item/weapon/reagent_containers))
+		return (locate(/obj/item/stack/sheet/mineral/clown) in holder.my_atom.contents) //you need that bananium catalyst
```

Comment:
Did you test this. Are you sure that grinding a sheet of bananium just adds it to the `holder.my_atom.contents`.
Besides, you could take inspiration from what how plasma works. For now I don't think bananium can even be ground up. When you have coded this, use 
```
required_catalysts = list(BANANIUM = 5)
``` 
or whatever

Reply:
![image](https://github.com/user-attachments/assets/e98a52b9-e118-4bbf-813a-b4ad6aa57ff9)
Reusing Midazoline code with a replaced item.